l2geth/core/genesis.go

```diff
@@ -547,7 +459,7 @@ func decodePrealloc(data string) GenesisAlloc {
 // fetchStateDump will fetch a state dump from a remote HTTP endpoint.
 // This state dump includes the OVM system contracts as well as previous
 // user state if the network has previously had a regenesis.
```

Comment:
I think its fine to fetch the state dump for now, I think longer term we could eliminate that but it would require some refactoring of the docker stack. Its possible that we could compile in the genesis state for each network using https://pkg.go.dev/embed and then utilize the network flags, ie `--kovan` to use the kovan state dump

Reply:
Mhm we definitely should switch to not fetching the dump but I'll do that as part of the same PR as the one that tackles your other comment above.